python/mlx/nn/layers/flatten.py

```diff
@@ -0,0 +1,61 @@
+import mlx.core as mx
+from mlx.nn.layers.base import Module
+
+
+def flatten(a: mx.array, start_dim: int = 0, end_dim: int = -1) -> mx.array:
+    r"""
+    Flattens a contiguous range of dims into a tensor.
+
+    See :class:`~nn.Flatten` for details.
+
+    Args:
+        input: input tensor.
+        start_dim: first dim to flatten (default = 0).
+        end_dim: last dim to flatten (default = -1).
+    """
+
+    return mx.flatten(a, start_dim, end_dim)
```

Comment:
Updated the python implementation to call `mx.flatten`

Reply:
This function should be removed. We don't need both `nn.flatten` and `mx.flatten`.